src/Instance/utils.js

```diff
@@ -57,3 +59,15 @@ export const navigateToHoldingsViewPage = (history, location, instance, holding,
     },
   });
 };
+
+export const extendKyWithTenant = (ky, tenantId) => {
+  return ky.extend({
+    hooks: {
+      beforeRequest: [
+        request => {
+          request.headers.set(OKAPI_TENANT_HEADER, tenantId);
+        },
+      ],
+    },
+  });
+};
```

Comment:
What's the purpose of this? [`@folio/stripes/core::useOkapiKy()` already supports a `tenant` prop](https://github.com/folio-org/stripes-core/blob/0cdfe80aab2418acafb008650a3671babe1f0d15/src/useOkapiKy.js). Is this something different or am I misunderstanding? It's hard to tell what your goal is when there aren't any explanatory comments.

Reply:
Although `useOkapiKy` supports a passed tenant, it is still a hook and imposes restrictions for hooks. We need the ability to specify the tenant for the HTTP client dynamically from the received data, in this case a react hook will not help, except as will create a basic HTTP client, which we can configure.

E.g. we have to update a piece in a tenant where it's located.
https://github.com/folio-org/ui-inventory/pull/2614/files#diff-4a37a8dce3623a54155ee5e8a9e5845b7c2f1957ef1bad4c7e919aecdd15c36fR13